Themis/Services/Assessment/AssessmentService.swift

```diff
@@ -11,13 +11,27 @@ import Common
 
 protocol AssessmentService {
     /// Delete all saved feedback and release the lock of the submission
-    func cancelAssessment(submissionId: Int) async throws
+    func cancelAssessment(participationId: Int?, submissionId: Int) async throws
```

Comment:
I guess the participation ID is necessary for the text assessment API?

Reply:
Yes. It may also be needed for other exercise types, such as modeling.